application/composer.json

```diff
@@ -18,7 +18,7 @@
     "silinternational/php-env": "^2.1.1",
     "silinternational/yii2-json-log-targets": "^1.0.0",
     "silinternational/yii2-email-log-target": "^1.0.1",
-    "silinternational/idp-id-broker-php-client": "^2.6.0",
+    "silinternational/idp-id-broker-php-client": "dev-develop",
```

Comment:
I'm guessing this will need to get changed before develop is merged into master.

Reply:
Yep, I'll need to be cognizant of that. Thanks.